new/LIT6939GadlaSosenna.xml

```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6939GadlaSosenna" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Gadla Sosǝnnā</title>
+                <title xml:lang="en" corresp="#t1">Life of Susanna</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The text belonging to the Beta ʾƎsrāʾel (Ethiopian Jewish) literature. It stands in close relation to the Christian Homily on the 
+                    Archangel Michael for Naḥase.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="BetaEsraelLiterature"/>
+                    <term key="Hagiography"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-02-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:isRelatedTo" active="LIT6939GadlaSosenna" passive="LIT3966Homily"/>
```

Comment:
the main relation should be to LIT3194Susanna  as it is a version of the same story, relations `isAbout `Susanna (PRS11599Susanna) and Michael would also be helpful

Reply:
Is saws:isRelatedTo the appropriate relation to relate to LIT3194Susanna?

Reply:
Please check the guidelines https://betamasaheft.eu/Guidelines/?q=saws&id=relations

Reply:
I found ecrm:P129_is_about, what might be a suitable relation.

Reply:
this is the one